aakulich/aakulich3.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>>
+        <META charset="utf-8">
+        <TITLE>Программа на Python</TITLE>
+    </HEAD>
+    <BODY>
+     <h1>HOW-TO: Программа на Python, Часть 2</h1>
+     <p><img title="Python" src="E:\programm\Icon-Python.png" align="right" alt="PythonSymbol"></p>
+     <ul>
+          <li><a href="aakulich2.html">Программа на Python — часть 1</a></li>
+          <li><a href="aakulich3.html">Программа на Python — часть 2</a></li>
+          <li><a href="aakulich4.html">Программа на Python — часть 3</a></li>
+     </ul>
+     <h2><a name="содержание">Содержание</h2></a>
+     <h3>>HOW-TO: Программа на Python, Часть 2</h3>
+     <ul>
+       <a href="#списки"><li>Списки</li></a>
```

Comment:
Why tag a outside li?

Reply:
i did it , but i dont have my files on the other computer